apps/frontend/src/components/questionary/questionaryComponents/SampleDeclaration/SamplesAnswerRenderer.tsx

```diff
@@ -61,15 +61,19 @@ const SamplesAnswerRenderer = ({
         samples={samples}
         onClick={(sample) => setSelectedSampleId(sample.id)}
       />
-      <InputDialog
-        maxWidth="sm"
+      <StyledDialog
+        maxWidth="lg"
```

Comment:
In quite a few places the small dialogues have been increased in size, is this intentional?

Reply:
Yeah, the modal size was too small. Let me know if you have any concerns. @TCMeldrum